easybuild/easyconfigs/f/FSL/FSL-6.0.2-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,77 @@
+# This build still relies on the following libraries (or part of them) shipped with FSL:
+#    GDCHART, libprob, libcprob, cprob, newran, newmat
+# Attempts to use externally built libraries failed. Worth trying again in the future...
+#
+# Some tools like icmp are missing since they are built with the fslpython_install.sh script
+# which locally installs Miniconda and creates a virtual environment. This is too messy and
+# an alternative solution is necessary to install them.
+# https://github.com/easybuilders/easybuild-easyconfigs/issues/7899
+# https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FslInstallation#imcp.2Fimglob.2Fimmv_errors_after_installation
+
+name = 'FSL'
+version = '6.0.2'
+versionsuffix = '-Python-3.7.2'
```

Comment:
change to: `versionsuffix = '-Python-%(pyver)s'`

Reply:
Done.